tests/unit-core/test-snapshot.c

```diff
@@ -423,22 +423,24 @@ main (void)
                                                                   literal_buffer_c,
                                                                   LITERAL_BUFFER_SIZE,
                                                                   true);
-    TEST_ASSERT (lit_c_buf_sz == 200);
+    TEST_ASSERT (lit_c_buf_sz == 271);
 
     static const char *expected_c_format = (
-                                            "jerry_length_t literal_count = 4;\n\n"
-                                            "jerry_char_t *literals[4] =\n"
+                                            "jerry_length_t literal_count = 5;\n\n"
+                                            "jerry_char_t *literals[5] =\n"
                                             "{\n"
-                                            "  \"Bb\",\n"
-                                            "  \"aa\",\n"
-                                            "  \"aaa\",\n"
-                                            "  \"xzy0\"\n"
+                                            "  \"\\x42\\x62\",\n"
+                                            "  \"\\x61\\x61\",\n"
+                                            "  \"\\x61\\x61\\x61\",\n"
+                                            "  \"\\x66\\x6F\\x20\\x6F\",\n"
+                                            "  \"\\x78\\x7A\\x79\\x30\"\n"
```

Comment:
Please don't convert the ascii characters. Always use two hex chars after \x, so `a..f` letters will not be accidentally consumed by the compiler.

Reply:
I cannot follow you, could you explain this with a bit more detail? The double quote character is an ascii character, but it will cause problems in C format, like in my example in the commit message. It must be converted to hex or escaped. I thought this is an easier solution than escaping only some special characters.

Reply:
The patch changed `"  \"Bb\",\n"` -> `\"\\x42\\x62\",\n`.